src/xdocs/checks/javadoc/javadocstyle.xml

```diff
@@ -198,6 +201,16 @@ public class Test {
      */
     private void methodWithValidCommentStyle() {}
 
+    // ok below, @return tag automatically inserts a period after the text
+    /**
+     * {@return {@code true} if this object
+     * has been initialized, {@code false} otherwise}
+     */
+    private boolean isInitialized()
+    {
+       return true;
```

Comment:
Please make it single line method to use less vertical space.
Please add this chunk to all other examples